bitnami/metallb/values.yaml

```
@@ -915,6 +915,124 @@ speaker:
   ## @param speaker.logLevel Sets the speaker log level. Does not work if the args are overridden
   ##
   logLevel: "info"
+  ## @section FRR Speaker
+  frr:
+    ## @param speaker.frr.enabled Whether to enable FRR mode
+    ## ref: https://metallb.universe.tf/concepts/bgp/#frr-mode
+    ##
+    enabled: false
+    image:
+      ## @param speaker.frr.image.registry [default: REGISTRY_NAME] MetalLB Speaker image registry
+      ## @param speaker.frr.image.repository [default: REPOSITORY_NAME/metallb-speaker] MetalLB Speaker image repository
+      ## @skip speaker.frr.image.tag MetalLB Speaker  image tag (immutable tags are recommended)
+      ## @param speaker.frr.image.digest MetalLB Speaker image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
+      ## @param speaker.frr.image.pullPolicy MetalLB Speaker image pull policy
+      ## @param speaker.frr.image.pullSecrets Specify docker-registry secret names as an array
+      ##
+      registry: docker.io
+      repository: metallb-frr
```

Did you mean `bitnami/metallb-frr`?

```suggestion
      repository: bitnami/metallb-frr
```